articles/install-guide/update.md

```diff
@@ -14,67 +14,102 @@ Learn how to update the Microsoft Quantum Development Kit (QDK) to the latest ve
+1. updating your existing Q# files and projects to align your code with any updated syntax
+2. updating the QDK itself for your chosen development environment 
 
-## Updating Q# Projects 
+## 1. Updating Q# Projects 
+
+Regardless of whether you are using C# or Python to host Q# operations, follow these instructions to update your Q# projects.
 
 1. First, install the latest version of the [.NET Core SDK 3.0](https://dotnet.microsoft.com/download) and run the following command in the command prompt:
-```bash
-dotnet --version
-```
- Verify the output is 3.0.100 or higher, then follow the instructions below depending on your setup.
+	```bash
+	dotnet --version
+	```
+Verify the output is `3.0.100` or higher, then follow the instructions below depending on your setup (Visual Studio, Visual Studio Code, or directly the command line).
 
-### In Visual Studio
+### Update Q# projects in Visual Studio
  
- 1. Update to the latest version of Visual Studio 2019, see [here](https://docs.microsoft.com/visualstudio/install/update-visual-studio?view=vs-2019) for instructions
- 2. Open your solution in Visual Studio
- 3. From the menu, select Build > Clean Solution 
- 4. [Update the target framework](https://docs.microsoft.com/visualstudio/ide/visual-studio-multi-targeting-overview?view=vs-2019#change-the-target-framework) in each of your .csproj files to netcoreapp3.0 (or netstandard2.1 if it is a library project)
- 5. Save and close all files in your solution
- 6. Select Tools > Command Line > Developer Command Prompt
- 7. For each project in the solution, run the following command:
- ```bash
- dotnet add [project_name].csproj package Microsoft.Quantum.Development.Kit
- ```
-If your projects use any other Microsoft.Quantum packages, run the command for these too. 
- 8. Close the command prompt and select Build > Build Solution (do *not* select Rebuild Solution, as rebuilding will initially fail)
-
-### In Visual Studio Code
+1. Update to the latest version of Visual Studio 2019, see [here](https://docs.microsoft.com/visualstudio/install/update-visual-studio?view=vs-2019) for instructions
+2. Open your solution in Visual Studio
+3. From the menu, select **Build** -> **Clean Solution**
+4. In each of your .csproj files, update the target framework to `netcoreapp3.0` (or `netstandard2.1` if it is a library project).
+	That is, edit lines of the form:
+	```xml
+	<TargetFramework>netcoreapp3.0</TargetFramework>
+	```
+	You can find more details on specifying target frameworks [here](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks).
+5. Save and close all files in your solution
+6. Select **Tools** -> **Command Line** -> **Developer Command Prompt**
+7. For each project in the solution, run the following command:
+	```bash
+	dotnet add [project_name].csproj package Microsoft.Quantum.Development.Kit
+	```
+	If your projects use any other Microsoft.Quantum packages, run the command for these too.
```

Comment:
@bettinaheim @cgranade  @bromeg , could you give me an example or two of what one of these other packages might be? 
I'd like to put an "(e.g. ...)" here to avoid inexperienced users thinking this means they have to run it for namespaces (Microsoft.Quantum.Intrinsic, etc.)

Reply:
I think this is if they are using additional libraries, so I think Microsoft.Quantum.Chemistry might be an example, but I'd like @bettinaheim  and @cgranade  to confirm